templates/azure/sql.html

```diff
@@ -2,199 +2,249 @@
+          <p>Introduction to patching regimes and high availability clustering for SQL Server on Ubuntu Pro</p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="is-muted" />
+          <p>
+            <a href="/engage/SQLServerWebinarPart1" class="p-button">Watch the webinar</a>
+          </p>
         </div>
       </div>
-      <div class="col-6 p-divider__block">
-        <div class="p-heading-icon--small">
-          <div class="p-heading-icon__header">
-            {{
-              image(
-              url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
-              alt="",
-              width="32",
-              height="28",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img"}
-              ) | safe
-            }}
-            <h3 class="p-heading--4 p-heading-icon__title">Datasheet</h3>
+      <div class="p-equal-height-row__col"
+           style="grid-column: span 6;
+                  padding-top: 0.5rem">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container u-hide--small">
+            <img class="p-image-container__image p-strip is-shallow u-no-padding--top"
+                 src="https://assets.ubuntu.com/v1/e6cadf87-Image.png"
+                 alt="" />
           </div>
-          <p>SQL Server on Ubuntu Pro for Azure datasheet</p>
-          <p><a href="https://assets.ubuntu.com/v1/8098b6c6-Canonical.SQL.Server.datasheet.pdf" class="p-button--positive">Download the datasheet</a></p>
         </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Introducing SQL Server on Ubuntu Pro part 2</h3>
+        </div>
+        <div class="p-equal-height-row__item">
+          <p>
+            Anatomy of the performance features of SQL Server on Ubuntu Pro, how to set up AD integration, and running SQL Server on Ubuntu Pro with persistent memory (PMEM) Azure instances.
+          </p>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="is-muted" />
+          <p>
+            <a href="/engage/SQLServerWebinarPart2" class="p-button">Watch the webinar</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Get more out of SQL Server on Ubuntu Pro for Azure</h2>
+      </div>
+      <div class="col">
+        <h3 class="p-heading--5">Tutorial</h3>
+        <p>Launch a VM with SQL Server on Ubuntu Pro for Azure</p>
+        <hr class="is-muted" />
+        <div class="p-section--shallow">
+          <p>
+            <a href="/tutorials/provision-an-ubuntu-virtual-machine-running-sql-server-in-azure"
+               class="p-button">Follow
+            the tutorial</a>
+          </p>
+        </div>
+        <hr class="p-rule--muted" />
```

Comment:
It would be better to use only one of the two hr muted classes (`is-muted` or `p-rule--muted`) within the same file. Last time I spoke with Vanilla the favorable class was `p-rule--muted`

Reply:
I am using p-rule--muted everywhere except right before buttons as it completely removes the distance between hr and a button

Reply:
fyi, in the future these two classes will do the same thing. We are just waiting on [this pr](https://github.com/canonical/vanilla-framework/pull/5241) to get merged